src/client/app/chat/chat.component.css

```diff
@@ -672,4 +675,105 @@ mm-prescription {
     overflow-x: hidden;
     border: 4px solid #e4dcdc;
     border-radius: 11px;
+}
+
+.spinner {
+    width: 40px;
+    height: 13px;
+    display: inline-block;
+    text-align: center;
+    position: relative;
+    top: 4px;
+    border-radius: 26px
+}
+
+.spinner>div {
+    width: 6px;
+    height: 6px;
+    position: relative;
+    bottom: 3px;
+    background-color: #333;
+    border-radius: 100%;
+    display: inline-block;
+    -webkit-animation: sk-bouncedelay 1.4s infinite ease-in-out both;
+    animation: sk-bouncedelay 1.4s infinite ease-in-out both;
+}
+
+.muted>div {
+    display: inline-block
+}
+
+.spinner .bounce1 {
+    -webkit-animation-delay: -0.32s;
+    animation-delay: -0.32s;
+}
+
+.spinner .bounce2 {
+    -webkit-animation-delay: -0.16s;
+    animation-delay: -0.16s;
+}
+
+@-webkit-keyframes sk-bouncedelay {
+    0%,
+    80%,
+    100% {
+        -webkit-transform: scale(0)
+    }
+    40% {
+        -webkit-transform: scale(1.0)
+    }
+}
+
+@keyframes sk-bouncedelay {
+    0%,
+    80%,
+    100% {
+        -webkit-transform: scale(0);
+        transform: scale(0);
+    }
+    40% {
+        -webkit-transform: scale(1.0);
+        transform: scale(1.0);
+    }
+}
+
+.lds-ring {
```

Comment:
What does lds stand for?

Reply:
may be loadspinner. i haven't designed that